hw/xilinx/synth/constraints/au250.xdc

```diff
@@ -800,153 +806,153 @@ set_property -dict {PACKAGE_PIN BD21 IOSTANDARD LVCMOS12       } [get_ports pcie
 # #                JEDEC Order   DQS ->  0  9  1 10  2 11  3 12  4 13  5 14  6 15  7 16  8 17
 # #                Xil MIG Order DQS ->  0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17
 # #
-# set_property -dict {PACKAGE_PIN AR36 IOSTANDARD SSTL12_DCI     } [get_ports c0_ddr4_adr[16]  ]; # Bank 42 VCCO - VCC1V2 Net "DDR4_C0_ADR16"   - IO_L23N_T3U_N9_42
```

Comment:
I would keep the default names for simplicity, instead of just swapping `channel index` and `ddr4 prefix`

Additioanlly, I would adapt our new `DEFINE_DDR4_PORTS` macro, to just take an integer in input and bind to this pins., namely 0/1/2/3 instead of ch0/ch1/ch2/ch3.